SUMMARY.md

```diff
@@ -65,4 +65,7 @@
 * [Xhosa (xh)](xh/README.md)
 * [Yoruba (yo)](yo/README.md)
 * [Zulu (zu)](zu/README.md)
-* [Example Style Guide](guidelines/example_en-us.md)
+* Guidelines
```

Comment:
Thanks for fixing this. Given the size of the list, we should put this at the beginning of the list